### PR TITLE
add release notes

### PR DESCRIPTION
**Pull request by qpetraroia**
Release notes for Jan 31st to Feb 7th. Please review.

https://github.com/Azure/AKS/issues/1314 and https://github.com/Azure/AKS/issues/1621 need to be verified if they work in 1.20.2 can anyone comment on this? 

Do not merge.

### CODE REVIEW

**Review by keikhara — COMMENTED** [review]
Calico version is 3.17.1 from 3.8.9. - sorry should be for next week 2/15~

**Comment by qpetraroia** [comment]
> Calico version is 3.17.1 from 3.8.9. - sorry should be for next week 2/15~

Noted!

**Comment by palma21** [comment]
@Azure/aks-leads ptal

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+This release is rolling out to all regions - ETA for conclusion 2021-02-17 for public cloud.
+
+### Announcements
+
+* The Azure Kubernetes Service [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) feature will be retired on May 31st, 2021.
+* Once GA AKS will default to its new [GPU specialized image](https://aka.ms/aks/specialized-gpu-image) as the supported option for GPU-capable agent nodes.
+* Starting on the week of 15 February 2021 (Azure China Cloud and Azure Government Cloud users will get this update in the following weeks), we will upgrade AKS clusters Calico network policy from Calico version v3.8.9 to v3.17.1. This upgrade will cause a breaking change to the default behavior of all-interfaces Host Endpoints. For customers that use Host Endpoints, and only these, this version brings a change. Please follow our [guidance](https://github.com/Azure/AKS/issues/2089) to apply the appropriate label and Global Network Policy if you want to keep the v3.8.9 default behavior of all-interfaces Host Endpoints.
+
+### Release Notes
+
```
Comment (palma21) [inline CHANGELOG.md:14]: Cluster Start/Stop GA (feature)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+* Preview Features
+  * AKS now supports Private Clusters created with a custom DNS zone (BYO DNS zone). Read more [here](https://docs.microsoft.com/azure/aks/private-clusters#configure-private-dns-zone)
+* Behavioral Change
+  * The process of creating a Persistant Volume will be delayed until a pod is created. Allowing the Persistant Volume to be created in the same zone as the pod.
```
Comment (palma21) [inline CHANGELOG.md:18]: *persistent

Comment (palma21) [reply CHANGELOG.md:18]: And be specific here you can link to our docs here:
https://docs.microsoft.com/en-us/azure/aks/azure-disk-csi#create-a-custom-storage-class

Which should be updated to the opposite now that the default behavior has been reversed.
@mlearned @zr-msft @Nickomang

**Review by palma21 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (qpetraroia) [reply CHANGELOG.md]: Resolved

File: CHANGELOG.md (hunk 1 above)
Comment (qpetraroia) [reply CHANGELOG.md:14]: Resolved

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+
+### Release Notes
+
+* Features
+  * Cluster Start/Stop is now [GA](https://docs.microsoft.com/azure/aks/start-stop-cluster).
+* Preview Features
+  * AKS now supports Private Clusters created with a custom DNS zone (BYO DNS zone). Read more [here](https://docs.microsoft.com/azure/aks/private-clusters#configure-private-dns-zone).
+* Behavioral Change
+  * The process of creating a Persistent Volume will be delayed until a pod is created. Allowing the Persistent Volume to be created in the same zone as the pod. Read more [here](https://docs.microsoft.com/azure/aks/azure-disk-csi#create-a-custom-storage-class).
```
Comment (palma21) [inline CHANGELOG.md:20]: Start with, the AKS default storage class behavior now will be to (...)

**Review by palma21 — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (qpetraroia) [reply CHANGELOG.md:20]: ackd. Will add this change with the GH issues.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,30 @@
...
 
+## Release 2021-02-08
+
+This release is rolling out to all regions - ETA for conclusion 2021-02-17 for public cloud.
+
+### Announcements
+
+* The Azure Kubernetes Service [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) feature will be retired on May 31st, 2021.
+* Once GA AKS will default to its new [GPU specialized image](https://aka.ms/aks/specialized-gpu-image) as the supported option for GPU-capable agent nodes.
+* Starting on the week of 15 February 2021 (Azure China Cloud and Azure Government Cloud users will get this update in the following weeks), we will upgrade AKS clusters Calico network policy from Calico version v3.8.9 to v3.17.1. This upgrade will cause a breaking change to the default behavior of all-interfaces Host Endpoints. For customers that use Host Endpoints, and only these, this version brings a change. Please follow our [guidance](https://github.com/Azure/AKS/issues/2089) to apply the appropriate label and Global Network Policy if you want to keep the v3.8.9 default behavior of all-interfaces Host Endpoints.
```
Comment (tomkerkhove) [inline CHANGELOG.md:11]: Can people opt-out of this or defer it?

**Pull request merged**